### PR TITLE
simple watchdog

### PR DESCRIPTION
**Pull request by nordbergmikael**

### CODE REVIEW

File: internal/pkg/application/watchdog/watchdog.go [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+	for {
+		select {
+		case <-done:
+			return
+		default:
+			ctx := context.Background()
+			tenants, err := w.app.GetTenants(ctx)
+			if err != nil || len(tenants) == 0 {
+				if err != nil {
+					w.log.Error().Err(err).Msg("failed to fetch tenats")
```
Comment (shestyf) [inline internal/pkg/application/watchdog/watchdog.go:53]: sp. "tenats" -> "tenants"

File: pkg/client/client.go [hunk 2]
```diff
@@ -172,6 +172,8 @@ func (dmc *devManagementClient) FindDeviceFromInternalID(ctx context.Context, de
 		return nil, err
 	}
 
+	log.Debug().Msgf("response from iot-device-mgmt: %s", string(respBody))
```
Comment (shestyf) [inline pkg/client/client.go:175]: do we want to keep debug here?

File: internal/pkg/infrastructure/repositories/database/database_test.go [hunk 3]
```diff
@@ -11,20 +11,20 @@ import (
 	"github.com/rs/zerolog"
 )
 
-const devices string = `devEUI;internalID;lat;lon;where;types;sensorType;name;description;active;tenant
-a81758fffe06bfa3;intern-a81758fffe06bfa3;62.39160;17.30723;water;urn:oma:lwm2m:ext:3303,urn:oma:lwm2m:ext:3302,urn:oma:lwm2m:ext:3301;Elsys_Codec;name-a81758fffe06bfa3;desc-a81758fffe06bfa3;true;default
-a81758fffe05e6fb;intern-a81758fffe05e6fb;62.39160;17.30723;;urn:oma:lwm2m:ext:3428;Elsys_Codec;name-a81758fffe06bfa3;desc-a81758fffe06bfa3;true;notdefault
+const devices string = `devEUI;internalID;lat;lon;where;types;sensorType;name;description;active;tenant;intervall
```
Comment (istyf) [inline internal/pkg/infrastructure/repositories/database/database_test.go:14]: interval

File: pkg/client/client.go (hunk 2 above)
Comment (istyf) [reply pkg/client/client.go]: that will get quite noisy I think ...

**Pull request merged**